README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/IBM/<repo>.svg?branch=master)](https://travis-ci.org/IBM/<repo>)
+[![Build Status](https://travis-ci.org/IBM/.svg?branch=master)](https://travis-ci.org/IBM/<repo>)
```

Comment:
this should be:

[![Build Status](https://travis-ci.org/IBM/snap-and-translate.svg?branch=master)](https://travis-ci.org/IBM/snap-and-translate)